front/src/modules/timesStops/types.ts

```diff
@@ -10,6 +10,18 @@ export type TimeExtraDays = {
   dayDisplayed?: boolean;
 };
 
+export type TimesStopsRowNew = {
+  id: string;
+  name: string;
+  ch: string;
```

Comment:
I think `secondaryCode` would be better than `ch` (very French)
```suggestion
  secondaryCode: string;
```